spec/services/pull_request/draw_reviewer_spec.rb

```diff
@@ -0,0 +1,24 @@
+describe 'PullRequest::DrawReviewer' do
+
+  context 'when there is a reviewer with smallest reviews count' do
+    let!(:user) { FactoryGirl.create(:user, issues: 10) }
+    let!(:another_user) { FactoryGirl.create(:user, issues: 11) }
+    let!(:yet_another_user) { FactoryGirl.create(:user, issues: 3) }
+
+    it 'should draw that person' do
+      expect(PullRequest::DrawReviewer.call).to eq(yet_another_user.github_login)
+    end
+
+  end
+
+  context 'when all users have the same amount of issues' do
+    let!(:user) { FactoryGirl.create(:user) }
+    let!(:another_user) { FactoryGirl.create(:user) }
+    let!(:yet_another_user) { FactoryGirl.create(:user) }
+
+    it 'should draw first person from the top' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.